pkg/cmd/pulumi/destroy.go

```diff
@@ -193,8 +193,8 @@ func newDestroyCmd() *cobra.Command {
 	cmd.PersistentFlags().BoolVar(
 		&showSames, "show-sames", false,
 		"Show resources that don't need to be updated because they haven't changed, alongside those that do")
-	cmd.PersistentFlags().BoolVar(
-		&skipPreview, "skip-preview", false,
+	cmd.PersistentFlags().BoolVarP(
+		&skipPreview, "skip-preview", "f", false,
```

Comment:
Just to make sure - I don't have to also pass `--yes` do I?

(found the issue - I guess I do but only if I include `--non-interactive` - which is weird? -- https://github.com/pulumi/pulumi/issues/4569).

Reply:
@lukehoban so skip-preview and yes have 2 separate purposes here - what I can do is to have a new parameter called force (-f shorthand) that means we don't even care what skip-preview and yes are doing here

What do you think?

Reply:
If I'm right on what #4569 is about - then I think we just should also go fix that issue - though it's mostly orthogonal to this.  I don't think we need to introduce another flag - I believe `--skip-preview`/`-f` is intended to skip the preview and not prompt the user (there is nothing to prompt the user about).  So #4569 is just a bug.